sparkpost/exceptions.py

```diff
@@ -5,17 +5,14 @@ class SparkPostException(Exception):
 class SparkPostAPIException(SparkPostException):
     "Handle 4xx and 5xx errors from the SparkPost API"
     def __init__(self, response, *args, **kwargs):
-        errors = None
         try:
             errors = response.json()['errors']
-            errors = [e['message'] +
-                      ' Code: ' + e.get('code', '') +
-                      ' Description: ' + e.get('description', '') +
-                      '\n'
+            error_template = "{message} Code: {code} Description: {desc} \n"
+            errors = [error_template.format(message=e['message'],
```

Comment:
yeah we could - though if message isn't there this is useless to users - I could go either way w/it